kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/KafkaUtils.java

```diff
@@ -45,9 +49,9 @@ public class KafkaUtils {
      * @param topicName the topic name
      * @param bootstrap the bootstrap
      * @param timeoutMilliseconds the timeout milliseconds
-     * @return the string
+     * @return the log of the pod
      */
-    public static String ConsumeMessage(String deployNamespace, String topicName, String bootstrap, int timeoutMilliseconds) {
+    public static String consumeMessage(String deployNamespace, String topicName, String bootstrap, int timeoutMilliseconds) {
```

Comment:
I think it would be better to replace `int timeoutMilliseconds` with `Duration timeout` as it leads to a more expressive api at the call sites. 

Not something that needs to be changed in this PR as it probably make sense to do so in a more global fashion.

Reply:
Sure, thanks!